website/content/api-docs/features/consistency.mdx

```diff
@@ -37,15 +64,140 @@ The three read modes are:
   values. Since this mode allows reads without a leader, a cluster that is
   unavailable will still be able to respond to queries.
 
-To switch these modes, either the `stale` or `consistent` query parameters
-should be provided on requests. It is an error to provide both.
+  ~> **Scaling read requests**: horizontally scaling Consul servers under
+  heavy request load will only help if most requests are `stale` reads.
+  This is because `stale` reads are the only requests that can be handled by
+  server agents other than the leader. Other requests will always be handled
+  by the leader no matter the number of Consul servers.
+  The most effective way to increase read scalability is to convert non-`stale`
+  reads to `stale` reads.
+  Only `stale` requests benefit from the enhanced read scalability provided by
+  [read replicas](/docs/enterprise/read-scale) and
+  [redundancy zones](/docs/enterprise/redundancy).
 
-Note that some endpoints support a `cached` parameter which has some of the same
-semantics as `stale` but different trade offs. This behavior is described in
-[agent caching feature documentation](/api/features/caching).
+### Intra-Datacenter Request Behavior
+
+The following diagrams show how the intra-datacenter request path varies
+per consistency mode and the relative trade-offs between level of consistency and performance.
+
+<Tabs>
+<Tab heading="Stale reads">
+
+[![stale consistency mode read request path](/img/consistency-modes/consistency-mode-stale.png)](/img/consistency-modes/consistency-mode-stale.png)
+
+</Tab>
+
+<Tab heading="Default reads">
+
+[![default consistency mode read request path](/img/consistency-modes/consistency-mode-default.png)](/img/consistency-modes/consistency-mode-default.png)
+
+</Tab>
+<Tab heading="Consistent reads, all writes">
+
+[![consistent consistency mode read, any write request path](/img/consistency-modes/consistency-mode-consistent.png)](/img/consistency-modes/consistency-mode-consistent.png)
+
+</Tab>
+</Tabs>
+
+### Cross-Datacenter Request Behavior
+
+When making a request across federated Consul datacenters, requests are forwarded from
+a local server to any remote server. Once in the remote datecenter, the request path
+is the same as a [local request with the same consistency mode](#intra-datacenter-request-behavior).
+
+[![consistency mode behavior across Consul datacenters](/img/consistency-modes/consistency-mode-cross-datacenter.png)](/img/consistency-modes/consistency-mode-cross-datacenter.png)
+
+## Usage
+
+### Consul DNS Queries: Defaults and Overriding
```

Comment:
I'm describing both the default and how to override it, so I don't think it can't be "Overriding Consul DNS Query Defaults". Maybe "Consul DNS Query Mode Default and Overriding"? Or just "Consul DNS Queries" since it's under the "Usage" heading?